serverless/event_sources/knative-event-sources.adoc

```diff
@@ -0,0 +1,31 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="knative-event-sources"]
+= Getting started with event sources
+include::modules/common-attributes.adoc[]
+:context: knative-event-sources
+toc::[]
+
+An _event source_ is a resource object, which is the link between an event producer and a _sink_ (or consumer) that receives those events. Currently, {ServerlessProductName} supports the following event source types:
+
+ApiServerSource:: Connects a sink to the Kubernetes API server.
+PingSource:: Periodically sends ping events with a constant payload. It can be used as a timer.
+
+SinkBinding is also supported, which allows you to connect core Kubernetes resources such as `Deployment`, `Job`, or `StatefulSet` with a sink.
+
+You can create and manage Knative event sources using the Developer perspective, the `kn` CLI, or by applying YAML files.
+
+.Prerequisites
+
+* You must have a current installation of xref:../../serverless/installing_serverless/installing-openshift-serverless.adoc#serverless-install-web-console_installing-openshift-serverless[{ServerlessProductName}], including Knative Serving and Eventing, in your {product-title} cluster. This can be installed by a cluster administrator.
+* You must create the `default` broker in the same namespace that the event source will be installed in. For more information, see xref:../../serverless/knative_eventing/serverless-using-brokers.adoc#serverless-using-brokers[Using brokers with Knative Eventing].
+* You must create or update a service account, role and role binding for event sources.
```

Comment:
the service account is needed here, b/c the ApiServerSources is accessing "k8s events".

The ping source, for instance does not need that, to give a counter example

Reply:
So only the ApiServerSource needs the service a/c + the broker?

Reply:
The APIServerSource needs the service account b/c it accesses k8s event, like

```
  resources:
  - apiVersion: v1
    kind: Event
```

see here: https://github.com/matzew/knative-eventing-samples/blob/master/01-source_service/020-k8s-events.yaml#L9-L11

The source does NOT need the service-account, b/c of the broker.  Generally the `APIServrSrc` needs a ServiceAccount, it is completely independent if it does sink to a `Service`, a `Channel` or a `Broker`.
* K

Reply:
I'm not saying it needs the service a/c _because_ of the broker, I'm just saying ApiServerSource needs both a service a/c and a broker, right?